### PR TITLE
home layout

### PR DESCRIPTION
**Pull request by EvonneNgXinYi**
Build a simple home page with header,  and also help page, about page and sign up page.

### CODE REVIEW

**Comment by EvonneNgXinYi** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = "Gamer Community"
```
Comment (chungpv-1008) [inline app/helpers/application_helper.rb]: i18n for this text

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<% provide :title, t(".about_title") %>
+<h1><%= t ".heading" %></h1>
+<p>
+  <%= t ".description_title" %>
+</p>
```
Comment (chungpv-1008) [inline app/views/static_pages/about.html.erb]: add endline for this file

**Comment by EvonneNgXinYi** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**